src/il_representations/scripts/il_train.py

```diff
@@ -189,6 +189,22 @@ def make_policy(observation_space,
         else:
             encoder = encoder_or_path
         assert isinstance(encoder, nn.Module)
+
+        # Sometimes we are not using FC layers in encoders, e.g., Jigsaw.
+        # Hence in il training we need to recover this part of the encoder.
```

Comment:
Is it possible to have some more information here? I'm not totally sure what this code does, or why it's needed.

Reply:
Added more comments just now.